src/cares_wrap.cc

```diff
@@ -205,7 +205,7 @@ static void ares_sockstate_cb(void* data,
   } else {
     /* read == 0 and write == 0 this is c-ares's way of notifying us that */
     /* the socket is now closed. We must free the data associated with */
-    /* socket. */
+    /* the socket. */
```

Comment:
@thefourtheye This is not part of this PR, I speculate that this is mixed from previous commit